requirements.txt

```diff
@@ -6,4 +6,5 @@ platformdirs>=3.11.0
 datasets>=2.14.6
 diskcache>=5.6.3
 graphviz>=0.20.3
-gdown>=5.2.0
\ No newline at end of file
+gdown>=5.2.0
+boto3>=1.34.133
```

Comment:
can we make this optional? i.e., without adding this to requirements.txt, can we just leave it to the import error handler (like we have for e.g., anthropic.py / openai.py etc., saying people who want to use bedrock models should install boto?

Reply:
For sure we can, I'm already managing the import error :) let me change that